### ISSUE TITLE
Itty bitty site logos in Storefront 2.1.1+

### ISSUE THREAD

**Issue by mindsize** [woothemes/storefront#472]
I've had this happen on a couple of sites now. 

![screenshot 2016-09-09 10 07 27](https://cloud.githubusercontent.com/assets/938513/18393921/1c6db58a-767d-11e6-84e9-c074d3e4fe32.png)

Confirmed in most recent release builds of Chrome (macOS), Firefox (macOS) and Safari (macOS / iOS on devices / orientations that exceed the breakpoint width).

It seems that the CSS below is the culprit:

```
@media (min-width: 768px) {
    .site-header .custom-logo-link img, .site-header .site-branding img, .site-header .site-logo-anchor img, .site-header .site-logo-link img {
    
        ...

        max-width: 100%
    }

...

}
```
I've been able to resolve it temporarily by setting max-width for these selectors back to none in my child theme. I've removed my fix temporarily so you can see it happening on the live site:

https://speeddemon.mindsize.me

I also feel like the margin between the logo and the menu has gotten bigger, even after the logo size issues have been resolved. Am I losing my mind?

**Comment by mindsize** [comment]
Do we need the max-width declaration here at all? The parent container has a max-width that should be constraining the image, anyway... but if someone wants to upload an image that's too wide and it breaks things, should we be fixing that in the CSS? Apart from eliminating it completely, `max-width: fit-content;` or `max-width: max-content;` seem to be the preferred solutions moving forward, but currently require prefixes in Firefox and Safari and don't work in IE / Edge. http://caniuse.com/#feat=intrinsic-width

@jameskoster, thoughts?

**Comment by mindsize** [comment]
Turns out the issue was with CSS minification, though I'm curious why one rule is minifying badly (100%) when the other is minifying without issue (fit-content). Very strange. For reference, the minification was coming from WP Rocket on this site. On the other site, it was Autoptimize.

**Comment by jameskoster** [comment]
Hmm, not entirely sure what the issue is here. The logo needs to have the `max-width: 100% declaration` (as do all images) to stop them breaking out of their container(s). That's a .org requirement so we can't change it.

It might be worth raising this with the plugin author, perhaps it's a known issue?

Also just double checking the latest versions of WP and Storefront are being used? And it's the core logo feature rather than the Jetpack (or other plugin) one?

**Comment by mindsize** [comment]
Latest versions of everything and using core site logo support. I'm trying to do a bit more research to see why that particular line is being re-minified strangely by both plugins. Obviously exclusion from minification is the solution, but not everybody will know to do that, so I'd like to find a solution, if at all possible. 

I think the fact that the overriding style is being added in a non-minified file is why it has been resolved in the child themes when minification is on. I'm going to try defining the exact same selectors and rules in the child theme shortly to see if it's just the re-minification that is causing this or if it is the property itself. I can't see it being the property, but I want to make sure I've covered all the bases. Time to do some testing!
